package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "ffc-ce-admin-web",
+  "description": "Digital service for calculation engine.",
```

Comment:
This should be different from the user front end.
`Admin interface for the calculation engine digital service`